### ISSUE TITLE
THE PROBLEM 4.4.1-6 is WRONG

### ISSUE THREAD

**Issue by OneirdyniaYi** [fool2fish/dragon-book-exercise-answers#172]
From the definition of the follow set, we know,1.follow(bterm) = first(bexpr') except ∈ 
and 2.bexpr' => ∈ follow(bterm) = follow(bexpr)
In a word, follow(bterm) = {$,),or}

**Comment by mansicer** [comment]
That's right. We can see a previous issue for correct answer: https://github.com/fool2fish/dragon-book-exercise-answers/issues/60
